platform/core/src/types/Command.ts

```diff
@@ -3,3 +3,10 @@ export interface Command {
   commandOptions?: Record<string, unknown>;
   context?: string;
 }
+
+/**
+ * This is the format used within many items for multiple commands
+ */
+export interface Commands {
+  commands: [];
+}
```

Comment:
I expected Commands be ` = Commandp[]` but it is an object that has commands which is an array of what?

Reply:
I have the same question as @sedghi

Reply:
I would say let's name it ToolBarServiceCommands then